cellprofiler/modules/clearborder.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+"""
+
+Clear objects connected to the label image border.
+
+"""
+
+import skimage.measure
+import skimage.segmentation
+
+import cellprofiler.module
+import cellprofiler.object
+import cellprofiler.setting
+
+
+class ClearBorder(cellprofiler.module.ImageSegmentation):
```

Comment:
I think it makes more sense for the input of this module to be objects, rather than an image. Easy to fix by inheriting from `cellprofiler.module.ObjectProcessing`. :)